packages/react/src/components/Slider/useSlider.ts

```diff
@@ -260,17 +321,21 @@ export const useSlider = (props: ISliderProps, ref: React.Ref<HTMLDivElement>) =
   const onMouseDownProp: {} = disabled ? {} : { onMouseDown: onMouseDownOrTouchStart };
   const onTouchStartProp: {} = disabled ? {} : { onTouchStart: onMouseDownOrTouchStart };
   const onKeyDownProp: {} = disabled ? {} : { onKeyDown: onKeyDown };
+  const onFocusProp: {} = disabled ? {} : { onFocus: onThumbFocus };
 
+  const lowerValueThumbRef = React.useRef<HTMLSpanElement>(null);
+  useComponentRef(props, lowerValueThumbRef, lowerValue, [lowerValue, value]);
```

Comment:
You shouldn't call `useComponentRef` twice... that sets up the `componentRef` prop.

Reply:
Oops. Forgot to clean that up